docs/platforms/aws-lambda/nodejs/index.md

```diff
@@ -0,0 +1,465 @@
+# AWS Lambda: Node.js
+
+AWS Lambda is a serverless compute service that lets you run code without provisioning or managing servers.  You can measure Metrics, Traces and Logs of AWS Lambda via Serverless Console.  
+
+# Tracing
+
+If you use Serverless Console's AWS Lambda Extension for Node.js, it will automatically trace your AWS Lambda invocations, providing a timeline rich with information we curated to help you assess, optimize and troubleshoot, as well as trace services and data that triggered your AWS Lambda functions.
+
+Before the handler of your AWS Lambda function runs, our Extension loads our SDK and performs the auto-instrumentation logic.  This is what it looks like:
+
+```javascript
+import serverless from 'serverless-sdk-aws-lambda'
+
+serverless.instrument(params)
+
+// Your handler is called next...
+```
+
+Every AWS Lambda function invocation instrumented with our Extension generates a Trace.  This Trace contains the following Spans, some of which are optional depending on the modules you use within your code.  We put great effort into enriching this Trace with Tags useful for debugging across code, AWS Services and more.
+
+Here is a table of contents of Spans we currently capture, in an example hierarchical format:
+
+* **aws.lambda**
+  * **aws.lambda.initialization**
+  * **aws.lambda.invocation**
+    * **express**
+      * **node.https**
+      * **node.http**
+      * **aws.sdk**
+
+# Spans
+
+## `aws.lambda`
+
+This is the parent Span (aka the Trace) for an AWS Lambda. It measures the combined lifecyle phases of AWS Lambda Initialization and Invocation, and any logic performed within the Invocation phase.
+
+Additionally, the duration of this Span is what AWS Lambda bills for, based on 1ms increments. Duration charges apply to initialization code that is declared outside of the handler in the Initiatlization phase, code that runs in the handler of a function during the Invocation phase, as well as the time it takes for code in any last running Extensions to finish executing during Shutdown phase.
```

Comment:
@medikoo I'm not sure we can, unfortunately.  I have omitted it for now.  It's also worth noting that it happens not with every invocation, and has a 300ms cap.  Therefore, this may be worth omitting.  Whereas Initialization and Invocation have a much larger performance, cost and end-user impact.